Cargo.lock

```diff
@@ -7225,6 +7221,59 @@ version = "0.4.0"
 source = "registry+https://github.com/rust-lang/crates.io-index"
 checksum = "712e227841d057c1ee1cd2fb22fa7e5a5461ae8e48fa2ca79ec42cfc1931183f"
 
+[[package]]
+name = "windows"
+version = "0.57.0"
+source = "registry+https://github.com/rust-lang/crates.io-index"
```

Comment:
Can we disable sysinfo default features to get rid of these windows crates?
https://github.com/GuillaumeGomez/sysinfo/blob/c821de4249a85f566766890345830d4b24a2dcb7/Cargo.toml#L18

Reply:
Unfortunately, it's hard to remove these features as the`System` has been limited by the features `cfg(features="system")`:
https://github.com/GuillaumeGomez/sysinfo/blob/e022ae4fd1d27c2d7159cad3e7018fc08e5b822d/src/lib.rs#L71-L82